woodwork/statistics_utils.py

```diff
@@ -29,6 +36,17 @@ def _get_describe_dict(dataframe, include=None, callback=None):
             - unit (str): unit of measurement for progress/total
             - time_elapsed (float): total time in seconds elapsed since start of call
 
+        extra_stats (bool): If True, will calculate a histogram and top values for numeric columns, top values
```

Comment:
Also, since the logic where we make that decision of whether or not to also calculate top values is a little hard to parse, can we add a comment above those lines explaining why we do or don't calculate it? (If I'm understanding correctly, something like "only calculate top values if there are more bins than possible values" might make sense)

Reply:
Updated the docstring and added a comment in the code to clarify the logic a bit